plugin/src/main/java/io/jenkins/plugins/casc/RootElementConfigurator.java

```diff
@@ -33,6 +35,14 @@
                 configurators.add(new DescriptorConfigurator(descriptor));
         }
 
+        configurators.sort(Comparator.comparingDouble(c -> {
```

Comment:
is it worth extracting this comparator somewhere? looks the same as your other PR?

Reply:
Potentially 😓
Did not think of that...

Let me merge the two PRs than.